examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -81,6 +81,8 @@ bool GenerateBdxUri(const Span<char> & fileDesignator, Span<char> outUri, size_t
 OTAProviderExample::OTAProviderExample()
 {
     memset(mOTAFilePath, 0, kFilepathBufLen);
+    queryImageBehavior = kRespondWithUpdateAvailable;
+    delayedActionTimeSec = 0;
```

Comment:
nit: follow convention in the file and do `mDelayedActionTimeSec`, `mQueryImageBehavior`